lib/events/Modal/modalOfferRequestEvents.js

```diff
@@ -16,7 +16,7 @@ if (Meteor.isClient) {
                 payment: "",
                 // By Default
                 status: "Pending",
-                destination: Listing.find({_id: id}).fetch()[0].creator_id
+                destination: [Listing.find({_id: id}).fetch()[0].creator_id, Meteor.userId() ]
```

Comment:
Line is too long.<br>Identifier 'creator_id' is not in camel case.<br>'Listing' is not defined.<br>'id' is not defined.<br>'Meteor' is not defined.